internal/engine/upper_test.go

```diff
@@ -128,7 +128,8 @@ type fakeBuildAndDeployer struct {
 
 	// Set this to simulate a container update that returns the container ID
 	// it updated.
-	nextBuildContainer container.ID
+	nextLiveUpdateContainerID    container.ID
+	nextDockerComposeContainerID container.ID
```

Comment:
`nextDockerComposeContainerID` should go in a different paragraph with a different comment

Reply:
done